web_src/js/features/repo-code.ts

```diff
@@ -142,13 +142,7 @@ export function initRepoCodeView() {
         });
       }
       selectRange($(linesEls), $(selectedEls), from ? $(from) : null);
-
-      if (window.getSelection) {
-        window.getSelection().removeAllRanges();
-      } else {
-        document.selection.empty();
-      }
-
+      window.getSelection().removeAllRanges();
```

Comment:
I can't find anything on MDN regarding `document.selection`, so I think it never existed.